Core/Core/UIViews/CoreWebView.swift

```diff
@@ -391,7 +402,8 @@ extension CoreWebView: WKUIDelegate {
         guard let from = linkDelegate?.routeLinksFrom else { return nil }
         let controller = CoreWebViewController()
         configuration.userContentController.removeAllUserScripts()
```

Comment:
if we are not going to re-run the setup, should we keep user scripts intact? We only removed them so it wouldn't crash re-adding them.

Reply:
Yeah I think that makes sense.